### PR TITLE
Error propagation

### PR DESCRIPTION
**Pull request by Nonius**

### CODE REVIEW

File: src/storage/mod.rs [hunk 1]
```diff
@@ -1,74 +1,84 @@
...
-        Error::PathError
-    }
+/// The `ErrorKind` variants that can be produced by the [`StorageBackend`] implementations.
+///
+/// [`StorageBackend`]: ./trait.StorageBackend.html
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Fail)]
+pub enum ErrorKind {
+    /// 450 Requested file action not taken.
+    ///     File unavailable (e.g., file busy).
+    #[fail(display = "450 Tranisent file not available")]
```
Comment (hannesdejager) [inline src/storage/mod.rs]: Typo `Tranisent`

Comment (Nonius) [reply src/storage/mod.rs]: Should be changed

**Comment by mdirkse** [comment]
LGTM

**Comment by hannesdejager** [comment]
Nice: 

Screenshot 2019-10-19 at 11 16 29

Can we merge? LGTM.

**Comment by Nonius** [comment]
Yes, merge it.

**Pull request merged**